source/common/quic/envoy_quic_client_stream.h

```diff
@@ -78,6 +78,8 @@ class EnvoyQuicClientStream : public quic::QuicSpdyClientStream,
   Http::ResponseDecoder* response_decoder_{nullptr};
 
   bool decoded_1xx_{false};
+
+  std::string upgrade_type_;
```

Comment:
nit: Comment, please. In particular, what are the possible values this might take and under what conditions might it be set vs empty.